cdap-explore/src/test/java/co/cask/cdap/explore/service/BaseHiveExploreServiceTest.java

```diff
@@ -138,24 +144,23 @@
 
   protected static Injector injector;
 
-  @ClassRule
-  public static TemporaryFolder tmpFolder = new TemporaryFolder();
-
-  protected static void initialize() throws Exception {
-    initialize(CConfiguration.create());
+  protected static void initialize(TemporaryFolder tmpFolder) throws Exception {
```

Comment:
how come it has to be passed in? isn't it get a new folder each time initialize is called?